src/components/Section/Relationships/RelationshipStatus/Cohabitant.jsx

```diff
@@ -58,7 +58,7 @@ export default class Cohabitant extends ValidationElement {
     const state = {
       Name: {},
       Birthdate: null,
-      BirthPlace: null,
+      BirthPlace: {country: undefined},
```

Comment:
```suggestion
      BirthPlace: {},
```

Don't need to add that because `this.props.BirthPlace.country` will be `undefined` against an empty object. Please make sure you have your IDE/editor setup with ESLint.